docs/dotnet/dotnet-connection-traces.md

```diff
@@ -0,0 +1,399 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: HTTP client and server
+--->
+
+# Semantic Conventions for HTTP client connection spans emitted by .NET
```

Comment:
qq  - do we need all the attributes (that are on http.client.request.time_in_queue) on that span? It'd be right under the request span which has everything available.

I'll add it with attributes and add a todo note for  now

Reply:
based on the offline discussion let's not add any attributes to wait-for-connection for the time being since it's a child of HTTP request which has all the details